kvs/src/main.rs

```diff
@@ -1,35 +1,36 @@
 #![feature(never_type)]
 
-use std::{collections::HashMap, sync::Arc, time::Duration};
+use std::time::Duration;
 
 use hydroflow::{
+    builder::{
+        prelude::{BaseSurface, PullSurface, PushSurface},
+        surface::pull_iter::IterPullSurface,
+        HydroflowBuilder,
+    },
     lang::{
         collections::Single,
-        lattice::{
-            dom_pair::DomPairRepr, map_union::MapUnionRepr, ord::MaxRepr, LatticeRepr, Merge,
-        },
+        lattice::{map_union::MapUnionRepr, ord::MaxRepr, set_union::SetUnionRepr},
         tag,
     },
+    scheduled::handoff::VecHandoff,
     tokio::{
         self,
-        sync::{
-            mpsc::{channel, Receiver, Sender},
-            Mutex,
-        },
+        sync::mpsc::{channel, Receiver, Sender},
     },
 };
 
 #[derive(Debug, Copy, Clone, Hash, Eq, PartialEq)]
 struct ActorId(u64);
 
-type Timestamp = HashMap<usize, u64>;
+// type Timestamp = HashMap<usize, u64>;
```

Comment:
Leave a comment to explain why this is commented out for now, and a reminder to come back and clean this up.